keyboards/totem/config.h

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022 GEIST <@geigeigeist>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Serial settings */
+#define USE_SERIAL
+/* serial.c configuration for split keyboard */
+#define SOFT_SERIAL_PIN GP0
+//#define SPLIT_LAYER_STATE_ENABLE
+#define SPLIT_USB_DETECT
+#define SERIAL_USE_MULTI_TRANSACTION
+/* communication between sides */
+#define SERIAL_PIO_USE_PIO1
+
+
```

Comment:
These are all either deprecated, the default behaviour, unneeded, or should be migrated to `keyboard.json`.
```suggestion
```

Reply:
I may be wrong here but these options maybe required for rp2040 specifically for it work in half-duplex mode because the firmware built without these options did not work for me previously. I'll try to flash my keyboard without these options to see if they are not required.